lib/signatures.ts

```diff
@@ -3,35 +3,40 @@ import { SignatureHashAlgorithm, SignatureResult } from './types';
 import { getDubHashSignature } from './dubhash';
 import { getUhashSignature } from './uhash';
 import pMap = require('p-map');
+import { join } from 'path';
 
 const { readFile } = promises;
 
+// Hash types supported by the signature module
```

Comment:
I don't know about this part. On one hand it's like a redefinition of the type in types.ts, but on the other hand it feels better to have enums or const when working with it. I will look into this a bit more